localization/camera/CMakeLists.txt

```diff
@@ -40,19 +32,17 @@ find_package(cmake_modules REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system)
 find_package(Eigen3 REQUIRED)
 
+# Find OpenCV
+SET(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../../cmake")
```

Comment:
I'm thinking maybe we should add this to catkin config --cmake-args, such that we never have to add this again..